actors/market/src/ext.rs

```diff
@@ -47,6 +52,67 @@ pub mod miner {
     pub struct IsControllingAddressParam {
         pub address: Address,
     }
+
```

Comment:
These types will move to the miner actor when we implement sending of the notification, hence initial definition in `ext` here.